pom.xml

```diff
@@ -35,16 +36,42 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <maven.test.failure.ignore>true</maven.test.failure.ignore>
   </properties>
 
   <build>
     <plugins>
+      <plugin>
+	    <groupId>org.codehaus.mojo</groupId>
+		<artifactId>jaxb2-maven-plugin</artifactId>
+		<version>3.1.0</version>
+		<executions>
+		  <execution>
+			<id>xjc</id>
+			<goals>
+			  <goal>xjc</goal>
+			</goals>
+		  </execution>
+		</executions>
+	    <configuration>
+		  <!-- The package of your generated sources -->
+		  <packageName>com.webcohesion.ofx4j.generated</packageName>
+		  <sources>
+			<source>src/main/resources/xsd</source>
+		  </sources>
+          <arguments>
+            <argument>-b</argument>
+            <argument>src/main/resources/jaxb-customization.xml</argument>
+          </arguments>
+	    </configuration>
+	  </plugin>
+    
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-compiler-plugin</artifactId>
         <configuration>
-          <source>1.6</source>
-          <target>1.6</target>
+          <source>17</source>
```

Comment:
I'd like to use the lowest common denominator to reduce upgrade friction. Can we get this to build in Java 8? If not at least Java 11?

Reply:
Have changed it in "1.8".